ozaria/engine/cinematic/CameraSystem.js

```diff
@@ -21,7 +27,26 @@ export class CameraSystem {
     const cameraMove = getCamera(shot)
     if (cameraMove) {
       const { pos: { x, y }, zoom } = cameraMove
-      commands.push(new SyncFunction(() => this.camera.zoomTo({ x, y }, zoom, 0)))
+      commands.push(this.zoomToCommand({ x, y }, zoom))
+    }
+    return commands
+  }
+
+  parseDialogNode (dialogNode) {
+    const commands = []
+    // Take a good guess at setting the camera to a reasonable default
+    // if it hasn't been set. We only set defaults for a zoom of 2.
+    if (this.lastCameraMove.zoom !== 2) {
+      return commands
+    }
+    if (this.lastCameraMove.pos.x !== 0 || this.lastCameraMove.pos.y !== 0) {
+      return commands
+    }
+    const speaker = getSpeaker(dialogNode)
+    if (speaker === 'left') {
+      commands.push(this.zoomToCommand({ x: -165, y: -65 }, 2))
+    } else if (speaker === 'right') {
+      commands.push(this.zoomToCommand({ x: 165, y: -65 }, 2))
```

Comment:
Do the values for x, y, and zoom here also come from the defaults? Should they be moved to constants in that case?

Reply:
Excellent idea!